src/main/java/com/meronat/iplog/IPLog.java

```diff
@@ -185,10 +192,21 @@ public Logger getLogger() {
 
     }
 
+    public PluginContainer getPluginContainer() {
+        return pluginContainer;
+    }
+
     public static IPLog getPlugin() {
 
         return plugin;
 
     }
 
+    public static <T> TaskChain<T> newChain() {
+        return factory.newChain();
+    }
+
+    public static <T> TaskChain<T> newSharedChain(String name) {
+        return factory.newSharedChain(name);
```

Comment:
Just read up on the topic, Aikar describes shared chains as something that you use when you want an operation to be nonblocking of the main thread, but it is blocking the object it's operating on. If at any point you feel like you would need this, then replace newChain calls in that scenario with newSharedChain(String) calls. If not, it can be removed